src/app/zap-templates/templates/app/cluster-objects-src.zapt

```diff
@@ -19,13 +19,7 @@ CHIP_ERROR FlightCheckDecodeAndEnterStruct(TLV::TLVReader & reader, TLV::TLVType
     CHIP_ERROR err = CHIP_NO_ERROR;
     VerifyOrReturnError(TLV::kTLVType_Structure == temp_reader.GetType(), CHIP_ERROR_WRONG_TLV_TYPE);
     ReturnErrorOnFailure(temp_reader.EnterContainer(outer));
-    while ((err = temp_reader.Next()) == CHIP_NO_ERROR)
-    {
-        if (!TLV::IsContextTag(temp_reader.GetTag()))
-        {
-            continue;
-        }
-    }
+    while ((err = temp_reader.Next()) == CHIP_NO_ERROR) {}
```

Comment:
Adding comment to not yet commit ... I would like to get CI validation and sizes as part of this PR. Overall I believe we may save a few KB still, but validating it.

Reply:
I believe the changes are now final - the iterator looks reasonably minimal to me and generic. May as well use this as one iteration step.